internal/types/staticctapi/staticctapi.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package types
+package staticctapi
```

Comment:
`staticctapi` is a bit "letter soupy", how about just `staticct` (or `static-ct`)?
Should be ok particularly since we're planning to rename this repo to `tesseract`...